css/styles.css

```diff
@@ -121,3 +120,98 @@ button {
     width: 100%;
   }
 }
+
+.hide {
+  display: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.dummy {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.dummy-row {
+  display: flex;
+  flex-direction: row;
+  margin: 30px;
+}
+
+.dummy-image {
+  width: calc(100% / 2);
+}
+
+.dummy-text {
+  width: calc(100% / 2);
+}
+
+.d-photo {
+  width: 90%;
+  height: 183px;
+}
+
+@media (max-width: 667px) {
+  .dummy-row {
+    display: flex;
+    flex-direction: column;
+    margin: 5px;
+  }
+
+  .dummy-text {
+    width: 100%;
+  }
+
+  li {
+    font-size: 10px;
+  }
+
+  h2.dummy {
+    font-size: 12px;
+  }
+
+  .dummy-image {
+    width: 100%;
+  }
+
+  .d-photo {
+    width: 100%;
+  }
+}
+
+.top-items {
+  display: flex;
+  flex-direction: row;
+}
+
+a {
+  padding-bottom: 15px;
+}
+
+.nav-elements {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 15px;
+}
+
+.entries {
+  display: flex;
+  justify-content: space-between;
+}
+
+.new-button {
+  background-color: white;
+  padding-top: 20px;
+}
+
+.new {
+  text-align: center;
+}
+
+.photo-label {
+  padding-top: 30px;
+}
```

Comment:
You should never have normal styling below a media query, please move this above your media query